### ISSUE TITLE
default user/password url node

### ISSUE THREAD

**Issue by andruwa13** [rancher/k3os#81]
https://localhost:6443 

default user & password ?

![https:localhost:6443 2019-05-09 19-04-47](https://user-images.githubusercontent.com/5846351/57468697-aa063700-728d-11e9-85ab-f617e127d2d4.png)

**Comment by immanuelfodor** [comment]
The user `rancher` can login without password on the Live CD running, but then you need to setup a new password. Have you tried it? Another idea is to login to the live CD and `sudo passwd rancher` to set up a new pass even before you install it with `sudo os-config`.

**Comment by immanuelfodor** [comment]
And then use here the pass that you set up - I left this sentence out :grinning:

**Comment by immanuelfodor** [comment]
Hmm, I set up a password after permanent install, and experience the same issue, the `rancher` user's credentials are not accepted here.

**Comment by xlight** [comment]
same here

**Comment by immanuelfodor** [comment]
Maybe we should not be able to login here at all?
I've found this in `sudo cat /var/log/k3s-service.log`

```
time="2019-05-14T05:40:45.144554887Z" level=info msg="Listening on :6443"
time="2019-05-14T05:40:45.608073256Z" level=info msg="Node token is available at /var/lib/rancher/k3s/server/node-token"
time="2019-05-14T05:40:45.608089801Z" level=info msg="To join node to cluster: k3s agent -s https://10.0.0.33:6443 -t ${NODE_TOKEN}"
```

I assume it's only useful for agents: https://github.com/rancher/k3os#k3ostoken

**Comment by jacobbaek** [comment]
I hope to help this someone.
you can see the username and password after running the 'cat /etc/rancher/k3s/k3s.yaml'.
at that time, print the username and password at the bottom.

**Comment by dweomer** [comment]
> same here.
> and ssh key import from Github not work either

this was fixed in #184 and #249 both included in the latest v0.6.0 release

**Comment by dweomer** [comment]
the password for an installed (or live) system can always be overridden with the `k3os.password` kernel parameter. you can edit this in grub by
- pressing "e" on the menu item
- adding `k3os.password=Y0urP4ssw0rd` to the end of the `linux` line
- hitting ctrl-x to save and boot with your configuration

please keep in mind that this is a per-boot override. so, after you login, it would be a good idea to change the password to something else
